config/mailinglist/rules/jdk.json

```diff
@@ -272,6 +272,8 @@
             "test/hotspot/jtreg/compiler/",
             "test/hotspot/jtreg/testlibrary/ctw/",
             "test/hotspot/jtreg/testlibrary/jittester/",
+            "test/hotspot/jtreg/testlibrary_tests/compile_framework/",
+            "test/hotspot/jtreg/testlibrary_tests/ir_framework/",
```

Comment:
Should we also add [generators/tests](https://github.com/openjdk/jdk/tree/master/test/hotspot/jtreg/testlibrary_tests/generators/tests)?

Reply:
Good catch, I added it as well.